### DISCUSSION TITLE
Using Excel Copy Buffer together with row selection for multi-deletes

### DISCUSSION THREAD

**Discussion by zewa666** [ghiscoding/slickgrid-universal#1592] [Q&A] [answered]
Hey there,

My grid is heavily built around the Excel Copy Buffer. Now trouble is users would still like to be able to select a couple of rows and delete  them in bulk. The page https://ghiscoding.gitbook.io/angular-slickgrid/grid-functionalities/row-selection although mentions that enableCheckboxSelector and enableExcelCopyBuffer do not work well together.

So for me that sounds like I'd have to add a custom bool column with checkboxes and handle that myself is that right @ghiscoding ?

**Comment by ghiscoding** [comment, marked as answer]
There was this SO that might be helpful, the guy tricked the system by toggling the selection model back and forth, it's more of a patch than anything though but the 2 separate selection models is the real reason why they can't be used together

https://stackoverflow.com/a/76810071/1212166

**Reply by zewa666** [reply]
ok thanks, that confirms my understanding that it should be handled in user land code.

**Comment by ghiscoding** [comment]
The new answer is to use the new Hybrid Selection Model that was brought in PR #2162 and can be demoed in [Example 37](https://ghiscoding.github.io/slickgrid-universal/#/example37) and [Example 38](https://ghiscoding.github.io/slickgrid-universal/#/example37), you can use it by enabling `enableHybridSelection: true` grid option
